react/DynamicIframe.tsx

```diff
@@ -11,8 +11,24 @@ const DynamicIframe: StorefrontFunctionComponent<DynamicIframeProps> = ({
   const {
     route: { params },
   } = useRuntime()
+
+  const { query } = useRuntime()
```

Comment:
You can combine this with the previous `useRuntime()` call like so:

`const {
  route: { params },
  query
} = useRuntime()`